src/components/eyewitness/eyewitness.tsx

```diff
@@ -99,41 +139,55 @@ export function Eyewitness() {
+     */
     const clearOutput = useCallback(() => {
         setOutput("");
         setHasSaved(false);
         setAllowSave(false);
     }, [setOutput]);
 
     return (
-        <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
-            {LoadingOverlayAndCancelButton(loading, pid)}
-            <Stack>
-                {UserGuide(title, description_userguide)}
-                <TextInput
-                    label={"Enter the file name or path containing URLs:"}
-                    placeholder={"Example: /home/kali/Desktop/filename"}
-                    required
-                    {...form.getInputProps("filepath")}
-                />
-                <TextInput
-                    label={"Enter the directory name where you want to save screenshots or define path of directory:"}
-                    placeholder={"Example: /home/kali/Directory name"}
-                    required
-                    {...form.getInputProps("directory")}
-                />
-                <TextInput label={"Enter the timeout time"} required {...form.getInputProps("timeout")} />
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <Button type={"submit"}>Scan</Button>
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-            </Stack>
-        </form>
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
+                {LoadingOverlayAndCancelButton(loading, pid)}
+                <Stack>
+                    <Title>{title}</Title>
+                    <p>{description}</p>
+                    <TextInput
+                        label={"Enter the file name or path containing URLs:"}
+                        placeholder={"Example: /home/kali/Desktop/filename"}
+                        required
+                        {...form.getInputProps("filePath")}
+                    />
+                    <TextInput
+                        label={
+                            "Enter the directory name where you want to save screenshots or define path of directory:"
+                        }
+                        placeholder={"Example: /home/kali/Directory name"}
+                        required
+                        {...form.getInputProps("directory")}
+                    />
+                    <TextInput label={"Enter the timeout time"} required {...form.getInputProps("timeout")} />
+                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
+                    <Button type={"submit"}>Scan</Button>
+                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
```

Comment:
Removed duplicate line.
```suggestion
                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
                    <Button type={"submit"}>Scan</Button>
```